### PR TITLE
fixed syntax error

### PR DESCRIPTION
**Pull request by danrot**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:+1:

File: src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function getImageAction(Request $request)
 
     /**
      * @param Request $request
-     * @param int $id
+     * @param int     $id
```
Comment (chirimoya) [inline src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php]: ???

File: src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php [hunk 2]
```diff
@@ -100,8 +100,8 @@ public function downloadAction(Request $request, $id)
 
     /**
      * @param FileVersion $fileVersion
-     * @param string $locale
-     * @param string $dispositionType
+     * @param string      $locale
```
Comment (chirimoya) [inline src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php]: ???

File: src/Sulu/Bundle/MediaBundle/Twig/MediaTwigExtension.php [hunk 3]
```diff
@@ -46,8 +46,8 @@ public function getFunctions()
     /**
      * resolves media id or object.
      *
-     * @param int|MediaEntity $media id to resolve
-     * @param string $locale
+     * @param int|MediaEntity $media  id to resolve
+     * @param string          $locale
```
Comment (chirimoya) [inline src/Sulu/Bundle/MediaBundle/Twig/MediaTwigExtension.php]: ???

File: src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php [hunk 4]
```diff
@@ -189,7 +189,7 @@ public function postAction(Request $request)
      * Change a filter by the given id.
      *
      * @param \Symfony\Component\HttpFoundation\Request $request
-     * @param int $id the attribute id
+     * @param int                                       $id      the attribute id
```
Comment (chirimoya) [inline src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php]: ???

File: src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php (hunk 2 above)
Comment (GrahamCampbell) [inline src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php]: phpdoc should be aligned

File: src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php (hunk 4 above)
Comment (GrahamCampbell) [inline src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php]: phpdoc should be aligned

Comment (GrahamCampbell) [inline src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php]: ALL your other phpdoc is already algined

Comment (danrot) [inline src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php]: Actually we don't do that, that's also the reason we've put that into our `.styleci.yml`: https://github.com/sulu-io/sulu/blob/develop/.styleci.yml#L10

If they are already aligned that was not intended. Has the configuration file maybe also changed?

Comment (dantleech) [inline src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php]: Also, shouldn't the namespace be "used" instead of expanded here?

Comment (GrahamCampbell) [inline src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php]: phpdoc_params was renamed to phpdoc_align

Comment (danrot) [inline src/Sulu/Bundle/ResourceBundle/Controller/FilterController.php]: @GrahamCampbell Ok, then I will adjust that :)
@dantleech Yeah, but that's a different topic. Or can PHPCSFixer or StyleCI also do that?

**Comment by danrot** [comment]
@chirimoya removed the aligning of phpdocs from the styleci config file.

File: src/Sulu/Bundle/ContactBundle/Tests/Functional/Controller/AccountControllerTest.php [hunk 5]
```diff
@@ -1256,12 +1256,12 @@ public function testPutNoDetails()
 
         $this->assertEquals('ExampleCompany', $response->name);
 
-        $this->assertEquals(0, sizeof($response->urls));
-        $this->assertEquals(0, sizeof($response->emails));
-        $this->assertEquals(0, sizeof($response->phones));
-        $this->assertEquals(0, sizeof($response->faxes));
-        $this->assertEquals(0, sizeof($response->notes));
-        $this->assertEquals(0, sizeof($response->addresses));
+        $this->assertEquals(0, count($response->urls));
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContactBundle/Tests/Functional/Controller/AccountControllerTest.php]: we should use here `assertCount` ... but this is not related to this PR

**Comment by wachterjohannes** [comment]
looks good @danrot

File: src/Sulu/Bundle/ContactBundle/Tests/Functional/Controller/AccountControllerTest.php (hunk 5 above)
Comment (danrot) [inline src/Sulu/Bundle/ContactBundle/Tests/Functional/Controller/AccountControllerTest.php]: Actually there is even a fixer doing that... But it is marked that it might change code behavior, that's why I don't really want to use it...
